package.json

```diff
@@ -24,7 +25,9 @@
     "lint": "eslint src",
     "flow": "flow check",
     "docs": "pushstate-server docs",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "tslint": "tslint typings/*.ts",
+    "typescript": "tsc ./typings/polished-test.ts --noEmit --target es6 --module es2015 --moduleResolution node"
```

Comment:
There's a range of things in typescript that can go wrong when using definitions when generating definitions.

Testing this adds a bit of extra complexity but is worthwhile imo.

https://github.com/paypal/glamorous/pull/124